frontend/src/Admin/pages/AssignJudges.js

```diff
@@ -3,6 +3,7 @@ import { Container } from 'reactstrap'
 import styled from 'styled-components'
```

Comment:
I'm not sure if my test data is in a weird state but when I go to the `judges/assign` page from the judges tab get a white page because of the this error: `Cannot read property '_subRows' of undefined`

Reply:
That's real strange. Can you give me more information so I can diagnose the problem? Browser & Version? Example Data? 

My first guess is `react-table` is having an issue rendering the data it's provided.

When I try visiting the "assign" page from that tab, everything works swimmingly in latest Chrome, Firefox, and Safari 10.1

Reply:
I can't repro this on my laptop, this section is good to go.